modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenOperation.java

```diff
@@ -29,7 +29,7 @@
             isArray, isMultipart,
             isResponseBinary = false, isResponseFile = false, hasReference = false,
             isRestfulIndex, isRestfulShow, isRestfulCreate, isRestfulUpdate, isRestfulDestroy,
-            isRestful, isDeprecated, isCallbackRequest, uniqueItems;
```

Comment:
Hi, could you elabore on what's the use case for this default response?

Reply:
I needed a way to determine if operation responses contain a default response. The way I'm handling each operation return value is to return an enum based on the response status code.